### DISCUSSION TITLE
Running arm64 after building it on amd64

### DISCUSSION THREAD

**Discussion by aclarknexient** [containers/podman#15741] [General]
Hello, I am trying to copy a workflow from my intel-based Linux box to my intel-based macbook pro. This is to build multi-arch manifest images for use by amd64 and arm64 archs.

On my Fedora 36 Linux box, I installed qemu-user-static, restarted its systemd service, and was able to run different architectures pretty easily:
```shell
$ platarch=linux/amd64,linux/arm64

$ podman build --jobs=2 --platform=$platarch --manifest shazam .

$ podman run --arch arm64 localhost/shazam arch
aarch64
$ podman run --arch amd64 localhost/shazam arch
amd64
```

The Dockerfile used by that build is very simple:
```docker
FROM debian:bullseye-slim
COPY README.md .
CMD ["cat /README.md"]
```

However, if I try to repeat those steps on my intel-based macbook pro, I can build OK but can't run:
```shell
$ uname -a
Darwin LT1234 21.6.0 Darwin Kernel Version 21.6.0: Wed Aug 10 14:25:27 PDT 2022; root:xnu-8020.141.5~2/RELEASE_X86_64 x86_64

$ podman run --arch arm64 localhost/shazam arch
WARNING: image platform ({amd64 linux  [] }) does not match the expected platform ({arm64 linux  [] })
{"msg":"exec container process `/usr/bin/arch`: Exec format error","level":"error","time":"2022-09-11T16:00:50.000336264Z"}
```

1. Do I need to install qemu-user-static into the Podman VM?
2. If so, how do I do that, or is there a built-in recipe/method to do that?
3. Is it possible to run linux/arm64 images on linux/amd64?

**Comment by mheon** [comment]
Convergint to discussion as this is a question. @baude PTAL

**Comment by afbjorklund** [comment]
* https://github.com/containers/podman/discussions/12899

**Reply by aclarknexient** [reply]
Thank you, that was helpful and I really appreciate that :)

From that discussion, I was able to install `qemu-user-static` in podman and run multi-arch containers/manifests:

```
$ podman machine ssh
$ sudo -i
$ rpm-ostree install qemu-user-static
$ systemctl reboot

$ podman run --arch arm64 alpine:latest arch
...output removed for brevity...
aarch64

$ podman run --arch amd64 alpine:latest arch
...output removed for brevity...
amd64
```
